src/Core/Infrastructure/RealTime/Api/FindService/FindServicePresenter.php

```diff
@@ -0,0 +1,84 @@
+<?php
```

Comment:
As you have created the Interface i don't think its mandatory to implement your Presenter in this PR.